tools/ci/docker/linux/Dockerfile

```diff
@@ -192,10 +192,11 @@ RUN cd /tools/renesas-tools/build/gcc && \
   make && make install
 
 # Install Rust and targets supported from NuttX
+ENV CARGO_HOME=/tools/rust
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
```

Comment:
I think we need to
```suggestion
RUN mkdir -p $CARGO_HOME && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
```

Reply:
but maybe `mkdir` is not needed.

Reply:
Tests on my local showed, that it not needed ... but let's add to make it clearly visible ... I'll update PR.